lms/lmsdb/models.py

```diff
@@ -726,6 +774,61 @@ def create_comment(
         return instance
 
 
+class Note(BaseModel):
+    creator = ForeignKeyField(User)
+    user = ForeignKeyField(User)
+    timestamp = DateTimeField(default=datetime.now())
+    note = ForeignKeyField(CommentText)
+    exercise = ForeignKeyField(Exercise, null=True)
+    privacy = CharField(choices=(
+        (NoteOptions.PRIVATE.value, NoteOptions.PRIVATE.value),
+        (NoteOptions.STAFF.value, NoteOptions.STAFF.value),
+        (NoteOptions.USER.value, NoteOptions.USER.value),
+        (NoteOptions.PUBLIC.value, NoteOptions.PUBLIC.value),
+    ))
+
+    @classmethod
+    def create_note(
```

Comment:
Move to models/notes.py

Reply:
Don't understand why.
It's same as Comment.